stdlib/importlib/metadata/__init__.pyi

```diff
@@ -69,8 +77,30 @@ class EntryPoint(_EntryPointBase):
     def __hash__(self) -> int: ...
     def __eq__(self, other: object) -> bool: ...
 
-if sys.version_info >= (3, 10):
-    class EntryPoints(list[EntryPoint]):  # use as list is deprecated since 3.10
+if sys.version_info >= (3, 12):
+    class EntryPoints(tuple[EntryPoint]):
```

Comment:
`tuple[EntryPoint]` means "a `tuple` of length 1, where the sole element is an instance of `EntryPoint`". Is that what you want here, or did you maybe mean

```suggestion
    class EntryPoints(tuple[EntryPoint, ...]):
```

Reply:
The latter. I've also got a stubtest failure, so I'll mark this as draft until I sort that out.